src/components/map/MapService.js

```diff
@@ -1308,6 +1313,18 @@ goog.require('ga_urlutils_service');
             easing: ol.easing.easeOut
           }));
           map.getView().setRotation(0);
+        },
+
+        adaptExtentPortalExtent: function(extent) {
```

Comment:
Maybe adapt function name to reflect better what it does? Like `mergeWithDefaultExtent`.